core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowResource.scala

```diff
@@ -459,14 +465,24 @@ class WorkflowResource {
   @Path("/search")
   def searchWorkflows(
       @Auth sessionUser: SessionUser,
-      @QueryParam("query") keywords: java.util.List[String]
+      @QueryParam("query") keywords: java.util.List[String],
+//      @QueryParam("page") page: Int = 1,
+//      @QueryParam("pageSize") pageSize: Int = 20,
+      @QueryParam("createDateStart") @DefaultValue("") creationStartDate: String = "",
+      @QueryParam("createDateEnd") @DefaultValue("") creationEndDate: String = "",
+      @QueryParam("modifiedDateStart") @DefaultValue("") modifiedStartDate: String = "",
+      @QueryParam("modifiedDateEnd") @DefaultValue("") modifiedEndDate: String = "",
+      @QueryParam("owner") owners: java.util.List[String] = new java.util.ArrayList[String](),
+      @QueryParam("id") workflowIDs: java.util.List[UInteger] = new java.util.ArrayList[UInteger](),
+      @QueryParam("operators") operators: java.util.List[String] =
```

Comment:
This is my fault... Let's change it to `operator`.

Reply:
`operators` is changed to `operator` now